src/extensions/default/InlineColorEditor/InlineColorEditor.js

```diff
@@ -97,25 +88,24 @@ define(function (require, exports, module) {
      * @return {?{start:{line:number, ch:number}, end:{line:number, ch:number}}}
      */
     InlineColorEditor.prototype.getCurrentRange = function () {
-        var start, end;
+        var pos, start, end;
         
-        start = this._startBookmark.find();
+        pos = this._marker.find();
```

Comment:
Yes, I'm certain that `this._marker` will never be `null` in the current code. But the constructor does not enforce that a valid marker is passed in and you can see some unit tests where creating `new InlineColorEditor()` does not pass any parameters (i.e. `marker` is `undefeined`). This is just to be safe for any future usage.